README.md

```diff
@@ -28,13 +29,13 @@ reading through another contributor's code.
 
 ### Types of Naming Conventions
 
-| Common Name   | Example                                        | Notes  |
-| ------------- | ---------------------------------------------- | ------ |
-| Pascal-case   | `DesertEagle`                                  |        |
-| Camel-case    | `desertEagle`                                  |        |
-| Snake-case    | `desert_eagle`, `Desert_Eagle`, `desert_Eagle` |        |

```

Comment:
[Wikipedia](https://en.wikipedia.org/wiki/Snake_case) states that *"the first letter of each word written in lowercase".*

Reply:
Sourcing Wikipedia gives me the shivers, but I admit my mistake.